elliottlib/model.py

```diff
@@ -1,5 +1,5 @@
 
-class ModelException(StandardError):
+class ModelException(Exception):
```

Comment:
... why?

Reply:
oh, this change was made by `autopep8` I use it to pass CI
the exception hierarchy tree is sth like this:
```
BaseException
 +-- SystemExit
 +-- KeyboardInterrupt
 +-- GeneratorExit
 +-- Exception
      +-- StopIteration
      +-- StandardError
```
So I guess `autopep8` decided to switch to `Exception` to help us catch more exceptions but base on our code is not that robust, I can change it back.